process/modules/common.py

```diff
@@ -13,6 +13,9 @@ def clipAreasByAreas(geometryToClip: gpd.GeoDataFrame, mask: gpd.GeoDataFrame, g
         mask_dissolved = mask
 
     mask_dissolved = mask_dissolved.explode(ignore_index=True)
+    mask_dissolved["geometry"] = mask_dissolved.make_valid()
+    mask_dissolved["geometry"] = mask_dissolved.normalize()
+    mask_dissolved.drop_duplicates()
```

Comment:
These three operations are done together repeatedly. Maybe they could be extracted to a function?

Reply:
Added own function for it